TechSupport-Session-Manager/src/main/java/com/github/RevatureTechSupport/TechSupportSessionManager/Repository/QueueRepository.java

```diff
@@ -0,0 +1,24 @@
+package com.github.RevatureTechSupport.TechSupportSessionManager.Repository;
+
+import com.github.RevatureTechSupport.TechSupportSessionManager.domain.Issue;
+import org.springframework.data.cassandra.repository.Query;
+import org.springframework.data.cassandra.repository.ReactiveCassandraRepository;
+import org.springframework.stereotype.Repository;
+import reactor.core.publisher.Mono;
+
+import java.util.UUID;
+
+@Repository
+public interface QueueRepository extends ReactiveCassandraRepository<Issue, UUID> {
+
+    @Query("SELECT * FROM issues WHERE inQueue= true AND reviewed = false ORDER BY openTime ASC limit 1 ALLOW FILTERING")
```

Comment:
Keep spacing between equals sign consistent
```suggestion
    @Query("SELECT * FROM issues WHERE inQueue = true AND reviewed = false ORDER BY openTime ASC limit 1 ALLOW FILTERING")
```